public/controllers/userController.js

```diff
@@ -98,86 +72,8 @@ angular.module('userController', ['userRegService'])
     }
   }
 
-  var validateRequired = function() {
```

Comment:
this function is removed as the required validation is checked using $valid of angular at signup page itself,so no need of an explicit required validation function